include/ofpi_arp.h

```diff
@@ -56,6 +65,7 @@ struct arp_entry {
 	odp_rwlock_t usetime_rwlock;
 
 	odp_bool_t is_valid;
+	union arp_entry_flags flags;
```

Comment:
The `is_valid` member could be moved inside `flags`.

Reply:
I was considering removing 'is_valid' as it has the same value with 'flags.is_complete' but ... this PR is an intermediary step and I am not convinced that I will not need it in the next PR. We need to better handle the relation with devices in the local network (maybe an extra route, etc.) in order to better handle and display incomplete arps for this case. My suggestion is to keep 'is_valid' as it is for now, until we figure out if we need it.

Reply:
Okay